src/plasma/plasma_manager.cc

```diff
@@ -244,17 +234,18 @@ struct PlasmaManagerState {
   const char *addr;
   /** Our port. */
   int port;
-  /** Hash table of outstanding fetch requests. The key is the object ID. The
+  /** Unordered map of outstanding fetch requests. The key is the object ID. The
    *  value is the data needed to perform the fetch. */
-  FetchRequest *fetch_requests;
+  std::unordered_map<ObjectID, FetchRequest *, UniqueIDHasher> *fetch_requests;
```

Comment:
these don't need to be pointers

we'll need to create/destroy the `PlasmaManagerState` with `new`/`delete`